docs/reports/requirements/1_proj-desc.md

```diff
@@ -65,34 +65,33 @@ Note: the User section will be color-coded to show what's already in place and w
+text-to-speech experience where an animated avatar reads user text aloud. The developers are the Kukarella employees
+who will be responsible for incorporating the API into their current system and adding more customizable assets for the
+avatars in the future.
 
-In the current Kukarella Table Reads TTS web application, the first step is for the user to create one or more actors.
+In the current Kukarella TTS web application, the first step is for the user to create one or more actors.
 The user gives the actor a name, selects a voice from a list of voices they can preview, selects a language from the
 list of approximately 150 languages, and selects or uploads an image of the actor. After creating the actor and
 entering the text they want to listen to, the user presses the play button, and then waits for the application to
 process and generate the audio.
 
-Once our API is incorporated into this system, if the user would like to both watch and listen to their avatar speak
-the provided text, the only difference in the user interface is that the user will select one of the "premium" avatar
-images for their actor. After pressing the play button, the user will wait for the Kukarella application and the API to
-process and generate a video of their avatar speaking the text with lips synchronized to the sounds produced from their text-to-speech API ```
-generated speech.
+Once our API is incorporated into this system, the enhanced experience of being able to both listen and watch the
+avatar speak will only be available with specific avatars. After pressing the play button, the user will wait for the
+Kukarella application and the API to process and generate a video of their avatar speaking the text with lips
+synchronized to the sounds produced in its speech.
 
 As for the developers, they will need to call the API within the Kukarella application. After receiving the user input
-and after the text is converted to speech in an audio file, the developer will call our API instead of providing the
-audio file as output for the user to listen to. When calling our API, the user-selected language, user-provided text,
-and the Kukarella-generated audio will be passed in, and then the API will provide the animated video as output. At
-this point, the Kukarella developer shows the video in the web application's user interface.
+and after the text is converted to speech in an audio file, the developer will call our API. When calling our API, the
+user-selected actor image, the user-selected language, the user-provided text, and the Kukarella-generated audio will
+be passed in, and then the API will provide the animated video as output. At this point, the Kukarella developer shows
+the video in the web application's user interface and provides the option to download the video file.
 
 When Kukarella decides to add more "premium" avatars, the developer will need to include the head, mouth, eye, and
```

Comment:
Change premium to select

Reply:
Good catch!! I forgot to adjust the last paragraph!